cellpack/test-recipes/v2/example.json

```diff
@@ -45,8 +45,9 @@
       "perturb_axis_amplitude": 0.1,
       "encapsulating_radius": 100,
       "packing": {
-        "mode": "random" // instead of "packing_mode": "random",
+        "packing_mode": "random" // instead of "packing_mode": "random",
```

Comment:
this should not be nested in `packing`. 
```
        "encapsulating_radius": 100,
        "packing_mode": "random" // instead of "packing_mode": "random",
```